utils/getClaimStatus.tsx

```diff
@@ -83,27 +83,41 @@ export function buildClaimStatusSummary(
   return buildTransLineProps(scenarioObject.summary, buildI18nKey(keys))
 }
 
+/**
+ * Get conditional continue certifying next step.
+ */
+export function displayContinueCertifying(): TransLineProps {
+  const keys = ['conditional-next-steps', 'continue-certifying']
+  return buildTransLineProps(claimStatusJson['conditional-next-steps']['continue-certifying'], buildI18nKey(keys))
+}
+
 /**
  * Get next steps content for Claim Status.
  */
 export function buildNextSteps(
   scenarioObject: ClaimStatusScenarioJson,
   scenarioString: string,
   whichSteps: StepType,
+  continueCertifying = false,
 ): TransLineProps[] {
   const steps: TransLineProps[] = []
   const json = scenarioObject[whichSteps]
   for (const [index, value] of json.entries()) {
     const keys = ['scenarios', scenarioString, whichSteps, index.toString()]
     steps.push(buildTransLineProps(value, buildI18nKey(keys)))
   }
+
+  // If we should display the "continue certifying" next step, then display it as the first step.
```

Comment:
Super minor, what about moving this above the for loop so it's being added in the desired resulting order (and using .push)? Seems slightly more readable?

Reply:
(side note, i really like the sound of the word `unshift`)